<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gestcommandes</class>
 <widget class="QDialog" name="Gestcommandes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:(:/img/ressources/img/bg.png)</string>
  </property>
  <widget class="QTabWidget" name="gestion">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>20</y>
     <width>1318</width>
     <height>700</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat ExtraBold</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#gestion{

background-image:(:/img/ressources/img/bg.png)}


QTabBar::tab {
  background: #6568F3;
  color: white;
  padding: 10px 12px;
border-radius:0px;
}

QTabBar::tab:selected {
    color: white;
   background-color:#FF7244;

}
QTabBar::tab:hover {
  background: #FF7244;
 }


QTabWidget::pane {
  top:0px; 
  background: rgb(245, 245, 245);
border-radius:10px;
} </string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_19">
    <attribute name="title">
     <string>Gestion des clients</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_20">
    <property name="styleSheet">
     <string notr="true">#tab_20{
background-image:url(:/img/ressources/img/bg.png);
}
</string>
    </property>
    <attribute name="title">
     <string>Gestion des commandes</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>371</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupBox_13{
background-color:white;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="descreption">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid #6568f3;
padding-left:7px;
border-radius:5px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_257">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_259">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Etat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_260">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Quantité Couleur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_261">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Quantité Sans Couleur</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#ajouter{
color:white;
background-color:#FF7244;
border-radius:10px;

}

#ajouter:hover{
background-color:#6568F3;
border-radius:10px;

}
</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_262">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#341763;
font-size:15px;</string>
      </property>
      <property name="text">
       <string>Ajouter une commande</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_263">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_264">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="eta">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid #6568f3;
padding-left:7px;
border-radius:5px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_266">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Qc">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid #6568f3;
padding-left:7px;
border-radius:5px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Qsc">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid #6568f3;
padding-left:7px;
border-radius:5px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_267">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>140</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_commande">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#clear_commande{
color:white;
background-color:#6568F3;
border-radius:10px;

}

#clear_commande:hover{
background-color:#FF7244;
border-radius:10px;

}
</string>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_commande">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#modifier_commande{
color:white;
background-color:#FF7244;
border-radius:10px;

}

#modifier_commande:hover{
background-color:#6568F3;
border-radius:10px;

}
</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_268">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat ExtraBold</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#4A5BCF;
font-size:15px;
</string>
     </property>
     <property name="text">
      <string>Listes des commandes</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_101">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>891</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#6568f3;
color:#4A5BCF;
padding:6px;
border-radius:10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_274">
     <property name="geometry">
      <rect>
       <x>1230</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat ExtraBold</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
</string>
     </property>
     <property name="text">
      <string>#</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_286">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>190</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Light</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#black;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_287">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>190</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Light</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#black;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="chercher_commande">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>20</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding:6px;
border-radius:10px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_commandeBtn">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_58{
color:white;
background-color:#FF7244;
border-radius:10px;



}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/search.png</normaloff>D:/search.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="valider_21">
     <property name="geometry">
      <rect>
       <x>1200</x>
       <y>300</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#valider_21{
color:white;
background-color:#6568F3;
border-radius:10px;

}

#valider_21:hover{
background-color:#FF7244;
border-radius:10px;

}
</string>
     </property>
     <property name="text">
      <string>Imprimer recu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_296">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>300</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat ExtraBold</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#4A5BCF;
font-size:15px;
</string>
     </property>
     <property name="text">
      <string>Trier par</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_7">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>300</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupBox_14{
background-color:white;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_297">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Heure</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_111">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid #6568f3;
padding-left:7px;
border-radius:5px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_298">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Jours</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_112">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid #6568f3;
padding-left:7px;
border-radius:5px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_299">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#341763;
font-size:15px;</string>
      </property>
      <property name="text">
       <string>Commande est préte dans</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>580</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#checkBox_7::indicator {
    width: 70px;
    height: 60px;
}

#checkBox_7::indicator:unchecked {
    image:url(:/img/ressources/img/eng.png);
}

#checkBox_7::indicator:checked {
    image: url(:/img/ressources/img/fr.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="horizontalFrame">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>340</y>
       <width>871</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_273">
        <property name="font">
         <font>
          <family>Montserrat ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>891</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>889</width>
        <height>199</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_269">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>60</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat ExtraBold</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
</string>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_270">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>60</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat ExtraBold</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
color:white;
</string>
     </property>
     <property name="text">
      <string>Etat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_271">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>60</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat ExtraBold</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
</string>
     </property>
     <property name="text">
      <string>Date commande</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_275">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>60</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat ExtraBold</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
</string>
     </property>
     <property name="text">
      <string>QS.Couleur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_272">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat ExtraBold</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
</string>
     </property>
     <property name="text">
      <string>Q.Couleur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshBtn">
     <property name="geometry">
      <rect>
       <x>1200</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#refreshBtn{
color:white;
background-color:#FF7244;
border-radius:10px;

}

#refreshBtn:hover{
background-color:#6568F3;
border-radius:10px;

}
</string>
     </property>
     <property name="text">
      <string>refreshBtn</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_21">
    <attribute name="title">
     <string>Gestions  des produits</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
